src/Illuminate/Validation/ConditionalRules.php

```diff
@@ -16,23 +16,23 @@ class ConditionalRules
     /**
      * The rules to be added to the attribute.
      *
-     * @var array|string|\Closure
+     * @var array|string|\Closure|\Illuminate\Contracts\Validation\ValidationRule|\Illuminate\Contracts\Validation\InvokableRule|\Illuminate\Contracts\Validation\Rule
```

Comment:
Please put the class first, then the scalars, thanks!

Reply:
As you prefer - I've changed the order.